tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -281,19 +281,19 @@ public byte[] toByteArray() {
    */
   private static class UnSyncByteArrayInputStream extends InputStream {
     protected byte[] buf;
-    protected int pos;
+    protected long pos;
```

Comment:
why change to long

Reply:
> why change to long

yes ,we can still use int, let me change it